website_sale_cart_decimals/static/src/js/variant_mixin.js

```diff
@@ -0,0 +1,26 @@
+/*  © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+    License LGPL-3.0 (http://www.gnu.org/licenses/lgpl-3.0.html)
+*/
+
+odoo.define('website_sale_cart_decimals.widgets', function (require) {
+    "use strict";
+
+    var variant_mixin = require('sale.VariantMixin');
+
+    variant_mixin.onClickAddCartJSON = function (ev) {
+        ev.preventDefault();
+        var $link = $(ev.currentTarget);
+        var $input = $link.closest('.input-group').find("input");
+        var min = parseFloat($input.data("min") || 0);
+        var max = parseFloat($input.data("max") || Infinity);
+        var previousQty = parseFloat($input.val() || 0, 10);
+        var quantity = ($link.has(".fa-minus").length ? -1 : 1) + previousQty;
+        var newQty = quantity > min ? (quantity < max ? quantity : max) : min;
+        newQty = Math.round(newQty * 100) / 100;
```

Comment:
Could you add a comment explaining that this line is actually the new one, and the function is fully copy & pasted from Odoo official addon?

Also, could you improve README file and point ot the commit version you based on, like here: https://github.com/solvosci/slv-stock/blob/13.0-add-sale_purchase_according_declared_stock/stock_move_action_assign_recode/README.rst ? If you we store this information, we'll update it later in a more secure way